content/components/toggle-switch.mdx

```diff
@@ -0,0 +1,145 @@
+        <Box as="p" mt="0">You may use a vertical layout when a toggle switch is layed out inline with other content, or if a horizontal layout would put the switch too far from its label. If the vertical layout is left-aligned, the "on"/"off" text may be moved to the right of the switch.</Box>
+    </div>
+    <Box display="flex" flexDirection="column" sx={{gap: 2, flexGrow: 1, flexShrink: 0}}>
+        <img
+            width="456"
+            role="presentation"
+            src="https://user-images.githubusercontent.com/2313998/159587115-5ffcdefb-f5e9-4180-8dc3-5328b0440f7f.png"
+        />
+        <img
+            width="456"
+            role="presentation"
+            src="https://user-images.githubusercontent.com/2313998/159587126-8d30ff26-33a8-4b84-87db-13dcfdff9dac.png"
+        />
+    </Box>
+</Box>
+
+### Size
+
+<Box mb={3} display="flex" alignItems="flex-start" flexDirection={["column", "column", "column", "column", "row"]} sx={{gap: 3}}>
+    <Box as="p" mt="0">A small variant is provided to maintain visual hierarchy, or for dense areas where the default size will not fit.</Box>
+    <img
+        width="456"
+        role="presentation"
+        src="https://user-images.githubusercontent.com/2313998/159587124-e7ef0a16-fd42-4614-be3e-80c985332ef7.png"
+    />
+</Box>
+
+### Loading
+
+<Box mb={3} display="flex" alignItems="flex-start" flexDirection={["column", "column", "column", "column", "row"]} sx={{gap: 3}}>
+    <Box as="p" mt="0">When a toggle switch is waiting for something to load before toggling, a loading indicator should be displayed. The switch should not prematurely toggle from "on" to "off". Wait until loading is complete before showing a change.</Box>
+    <img
+        width="456"
+        role="presentation"
+        src="https://user-images.githubusercontent.com/2313998/159587116-aa07a847-a633-4515-82b3-20ebe2349914.png"
+    />
+</Box>
+
+### Progressive disclosure
+
+<Box mb={3} display="flex" alignItems="flex-start" flexDirection={["column", "column", "column", "column", "row"]} sx={{gap: 3}}>
+    <Box as="p" mt="0">The <a href="/ui-patterns/progressive-disclosure">progressive disclosure pattern</a> may be used to hide or show content based on whether a toggle switch is on.</Box>
+    <img
+        width="456"
+        role="presentation"
+        src="https://user-images.githubusercontent.com/2313998/159591632-f604ec1d-aca9-42f7-ad65-16f8811f192f.png"
+    />
+</Box>
+
+## Accessibility
+
+### Keyboard navigation
+
+Toggle switches should have the same keyboard navigation features as a button. A toggle switch may be focused using the `Tab` key, and pressing the `Enter` or `Space` key will toggle the switch.
+
+### Touch target
+
+When the UI is rendered on a touch-screen device, ensure that the toggle switch has a large enough touch target size. The toggle switch should react when the user taps or hovers anywhere in the touch target area, even if it isn't directly on the control.
```

Comment:
"Toggle switch should react when the user taps or hovers anywhere in the touch target area". I'm thinking you mean the cursor should switch to a clickable cursor but it almost sounds like it will switch on hover. Maybe we can clarify?"

My suggestion doesn't sound quite right just yet but something similar? 

```suggestion
When the UI is rendered on a touch-screen device, ensure that the toggle switch has a large enough touch target size. The toggle switch should be able to be activated when the user taps or hovers anywhere in the touch target area, even if it isn't directly on the control.
```